tokio-console/args.example

```diff
@@ -36,8 +36,9 @@ OPTIONS:
             application runs interactively, stderr should generally be redirected to a file to avoid
             interfering with the console's text output.
             
-            [env: RUST_LOG=]
             [default: off]
+            
+            [env: RUST_LOG=]
```

Comment:
my assumption here is that the extra line of whitespace is inserted by `clap` and we can't get rid of it?

Reply:
Yes, you're correct. I have tried to remove the extra line but It is difficult.

Reply:
okay, i wouldn't worry about it too much, then. thanks!